math/realtime_stats.cpp

```diff
@@ -35,7 +35,7 @@ class stats_computer1 {
         n++;
         T tmp = x - K;
         Ex += tmp;
-        Ex2 += tmp * tmp;
+        Ex2 += (int64_t)tmp * tmp;
```

Comment:
This is a C-style type-cast. Instead, in C++, use the function style of typecast
```suggestion
        Ex2 += int64_t(tmp) * tmp;
```

Reply:
> This is a C-style type-cast. Instead, in C++, use the function style of typecast
> 
> or even better:

However, this would be incorrect. Since `tmp` could be an `int` or even a `float` in which case this would be erroneous